mappings/net/minecraft/client/render/entity/EntityRendererOcelot.mapping

```diff
@@ -1 +1,9 @@
 CLASS none/bua net/minecraft/client/render/entity/EntityRendererOcelot
+	FIELD a CAT_BLACK_SKIN Lnone/kp;
+	FIELD b OCELOT_SKIN Lnone/kp;
+	FIELD k CAT_RED_SKIN Lnone/kp;
+	FIELD l CAT_SIAMEASE_SKIN Lnone/kp;
```

Comment:
Alternative suggestion: `WILD_SKIN` / `TAMED_<color>_SKIN`. (Similar to wolves.)